editor/src/components/inspector/sections/layout-section/self-layout-subsection/self-layout-controls.tsx

```diff
@@ -27,7 +27,6 @@ export const LayoutTypePicker = React.memo(({ value, setActiveTab }: LayoutTypeP
           break
         }
         default: {
-          const _exhaustiveCheck: never = newValue
```

Comment:
We need this please. This line is to ensure we catch changes made to the type `SelfLayoutTab` so that we can update the switch statement. It looks like both of the new types should be grouped with "absolute" and "sticky", calling `onSubmitValue`